features/absolute-positioning.yml

```
@@ -0,0 +1,6 @@
+name: Absolute positioning
+description: "The `position: absolute` CSS declaration removes an element from the normal flow and positions it relative to its containing block or closest positioned ancestor."
```

`containing block` is a term with very little developer understanding, but I could not find a better one while working with anchor positioning. Flagging in case anyone else has one.

```suggestion
description: "The `position: absolute` CSS declaration removes an element from the normal flow and positions it relative to its containing block, which is often the root element, or closest positioned ancestor."
```